### ISSUE TITLE
Cumulative Reward suddenly dropped significantly.

### ISSUE THREAD

**Issue by moonsh** [Unity-Technologies/ml-agents#2007]
![image](https://user-images.githubusercontent.com/13994270/57184004-a421e780-6e82-11e9-9a71-d33a47462acb.png)

I didn't change anything during the training but the cumulative reward was suddenly dropped.

Could you tell me what problems cause this?

Thank you,

**Comment by dracolytch** [comment]
That's pretty unusual, and also unusual is that it looks like it's right at the 500,000 steps mark. Can you tell more about the problem/config? Are you using an academy or similar?

**Comment by moonsh** [comment]
@dracolytch Thank you for the response. I already handled this issue. After I normalized the input? (I turned the normalization on)  it becomes fine.  However, I can't understand this.

**Comment by djsdvg** [comment]
Hi, I have the same issues
![Schermata 2019-05-26 alle 09 42 51](https://user-images.githubusercontent.com/29379818/58378822-c814ac80-7f9a-11e9-8b75-98afbbfb005e.png)
Here's a screen of the reward. It's very strange because after a decent learning the reward suddenly falls down.
And the Entropy is strange too !
![Schermata 2019-05-26 alle 09 43 01](https://user-images.githubusercontent.com/29379818/58378841-0e6a0b80-7f9b-11e9-84b6-6f0df7c8f103.png)
When the rewards went to zero the entropy was zero too. This make no sense to me.
How can the train forget everything it learns before ?
Why after its fall it didn't get up again ?

I trained with ``num.envs = 64``and with this parameters:
```
batch_size = 2048
buffer_size = 131072
hidden_units = 512
learning_rate = 0.0005
max_steps 8.0e6
num_epoch 5
num_layers 3
time_horizon 1024
```

**Comment by ScriptBono** [comment]
I have a smiliar problem. After some time the performance suddenly drops significantly and stays low. Can't explain where this comes from. I'm using curriculum learning and used four environments in parallel.

![image](https://user-images.githubusercontent.com/4007801/58487515-fbf0fd00-8167-11e9-9227-f8cd760b2171.png)

![image](https://user-images.githubusercontent.com/4007801/58487797-833e7080-8168-11e9-9162-5242db9d4cd6.png)

**Comment by unityjeffrey** [comment]
Thanks for reaching out to us. We are closing this due to inactivity, but if you need additional assistance, feel free to reopen the issue.
